server/lib/validation/languages/tok.js

```diff
@@ -0,0 +1,63 @@
+// Amount of characters allowed per sentence to keep recordings in a manageable duration.
+const MIN_LENGTH = 1;
+const MAX_LENGTH = 90;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    return sentence.length < MIN_LENGTH || sentence.length > MAX_LENGTH;
+  },
+  error: `toki sitelen ni li suli ike, li ken jo e sitelen lili pi mute ni taso: ${MIN_LENGTH}-${MAX_LENGTH}`,
+}, {
+  regex: /[0-9]+/,
+  error: 'nanpa o lon ala toki ni',
+}, {
+  regex: /[<>+*#@%^[\]()/]/,
+  error: 'sitelen nasa o lon ala toki ni',
+}, {
+  // capital letters at start of word only
+  regex: /[\w\.][A-Z]/,
```

Comment:
The `.` does not need to be escaped within `[ ... ]`.

Reply:
Oops. Didn't know that.
I changed it to `\w[A-Z]|[A-Z]\.[A-Z]`, which is also more clear as to what the `\.` was for.

Reply:
Yes, that is indeed clearer now.